### DISCUSSION TITLE
Can't access replays via remote access

### DISCUSSION THREAD

**Discussion by patchido** [blakeblackshear/frigate#14489] [Ask A Question]
I need some help understanding what a problem can be caused by.

This only happens while used remotely. I use the cloudfare to do this

I am running frigate(latest) addon, when trying to replay video of my reolink doorbell for some reason it asks me for a user and password(frigate) by putting one that I usually use it redirects me to frigate main screen and it does that over and over. 

If I try to use the replay of other cameras( wyze over rtsp) it works just fine. What can be causing this?

**Comment by NickM-27** [comment]
are you trying to connect to port 8971? this seems to indicate cookies are disabled or something

**Reply by patchido** [reply]
i wouldnt know, im connecting through the home assitant companion app, it has the frigate addon installed and from there im trying to view

**Reply by NickM-27** [reply]
but that doesn't involve cloudflare then, we need to know how you have things setup

**Reply by patchido** [reply]
i have the cloudflare addon on HomeAssistant, i bought a domain name which I added the DNS record on my cloudflare account so that the dns would point to the cloudflare addon tunnel. im not that of an expert here, what else can i check on how i set it up? i did this following a guide
